package.json

```diff
@@ -28,16 +28,15 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@hapi/boom": "7.x.x",
-    "db-errors": "0.2.x"
+    "@hapi/boom": "9.x.x"
```

Comment:
Just a note that this version of boom (also lab and code) technically only supports node v12, and this plugin seems to officially support node v8+ (based on the travis config).  Whatever you want to do with that info is cool with me!